src/Microsoft.CodeAnalysis.Testing/Directory.Build.props

```diff
@@ -8,7 +8,12 @@
   </PropertyGroup>
 
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
-
+  
+  <PropertyGroup Condition="'$(CopyrightMicrosoft)' != ''">
```

Comment:
`Condition="'$(CopyrightMicrosoft)' != ''"` shouldn't be necessary anymore. Arcade now defines the property globally.

Reply:
so just this?

```xml
  <PropertyGroup>
    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
  </PropertyGroup>
```

Reply:
No, include the Copyright, just not the condition.